packages/dev/core/src/Maths/math.color.ts

```diff
@@ -29,7 +30,9 @@ function colorChannelToGammaSpaceExact(color: number): number {
 /**
  * Class used to hold a RGB color
  */
-export class Color3 {
+export class Color3 implements VectorLike<[number, number, number]> {
+    public readonly dimension: [number] = [3];

```

Comment:
Why is this one needed ? and as an array ?

Reply:
> Why is this one needed ? 

Do you mean `dimension`? `dimension` allows code to determine how many components the `VectorLike` object has. 

> as an array ?

Because the mathematical tensor dimension of a `VectorLike` object is `[N]`, where `N` is the number of components in the vector.